doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -679,7 +679,15 @@ Bluetooth libraries and services
 Bootloader libraries
 --------------------
 
-|no_changes_yet_note|
+* :ref:`mcuboot_ncs`:
+
+  * Allow the path in :kconfig:option:`CONFIG_BOOT_SIGNATURE_KEY_FILE` to be relative to the current sample directory.
+    See the help text of the kconfig option for more details.
+
+* :ref:`bootloader`:
+
+  * Fixed: When both :ref:`bootloader` and :ref:`mcuboot_ncs` are present, and MCUboot is provided as a hex file (via :kconfig:option:`CONFIG_MCUBOOT_BUILD_STRATEGY_USE_HEX_FILE`), the provision page was not regenerated correctly.
+    Fix generation of the provision page, and also add the option to provide the provision page as a hex file or, be skipped (:kconfig:option:`CONFIG_PROVISION_BUILD_STRATEGY_USE_HEX_FILE` and :kconfig:option:`CONFIG_PROVISION_BUILD_STRATEGY_SKIP_BUILD`)
```

Comment:
```suggestion
  * Added:

    * the :kconfig:option:`CONFIG_PROVISION_BUILD_STRATEGY_USE_HEX_FILE` Kconfig option to provide the provision page as a HEX file.
    * the :kconfig:option:`CONFIG_PROVISION_BUILD_STRATEGY_SKIP_BUILD` kconfig option to skip the building of the provision page. 
 
  * Fixed:

    * The generation of the provision page.
    * The regeneration of the provision page when both :ref:`bootloader` and :ref:`mcuboot_ncs` are present, and MCUboot is provided as a HEX file (using :kconfig:option:`CONFIG_MCUBOOT_BUILD_STRATEGY_USE_HEX_FILE`).
```

This is a guess, let me know if the descriptions of the options mentioned are correct.